ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -892,7 +909,7 @@ extension DrawerController: UIViewControllerTransitioningDelegate {
             if #available(iOS 13.0, *) {
                 useNavigationBarBackgroundColor = (direction.isVertical && source.traitCollection.userInterfaceLevel == .elevated)
             }
-            return DrawerPresentationController(presentedViewController: presented, presenting: presenting, source: source, sourceObject: sourceView ?? barButtonItem, presentationOrigin: presentationOrigin, presentationDirection: direction, presentationOffset: presentationOffset, presentationBackground: presentationBackground, adjustHeightForKeyboard: adjustsHeightForKeyboard, shouldUseWindowFullWidthInLandscape: shouldUseWindowFullWidthInLandscape)
+            return DrawerPresentationController(presentedViewController: presented, presenting: presenting, source: source, sourceObject: sourceView ?? barButtonItem, presentationOrigin: presentationOrigin, presentationDirection: direction, presentationOffset: presentationOffset, presentationBackground: presentationBackground, adjustHeightForKeyboard: adjustsHeightForKeyboard, shouldUseWindowFullWidthInLandscape: shouldUseWindowFullWidthInLandscape, delegateeTouchView: delegateeTouchView)
```

Comment:
why is delegateeeTouchView needed? can't we just use TouchForwardingView to forward the touch but not handling touches within the presentationcontroller? this will help us handle darkening the background when bottomsheet is expanded?

Reply:
delegateeTouchView is required so that Presented view can decide where the touch events should get delegated (NavigationViewController root view or presented view controller root view etc).

**can't we just use TouchForwardingView to forward the touch but not handling touches within the presentationcontroller?**

Can you please elaborate?  Are you asking to use TouchForwardingView instead of DelegatorTouchView? In this case, we want to delegate the touch events from outside tappable area of the drawer to the delegateeTouchView passed by presentedController. DelegateeTouchView also should get the touch events in the respective subview where the user tries to touch.

**this will help us handle darkening the background when bottomsheet is expanded?**
Right now in case of passThrough background, we want the background to be transparent.

Reply:
I saw design mocks of bottom sheet that when it is shown in its full extent it shows dark background. My suggestion is that control will have its passthrough uiview. TouchForwardingUIView by default doesn't handle "open override func point(inside point: CGPoint, with event: UIEvent?) -> Bool" in theory, whatever UIView that is shown by the client should receive the event.

Reply:
Did changes for same. Please check.